src/webapp-lib/doc/latex-editor.pug

```diff
@@ -35,11 +35,12 @@ block content
         img(src="https://storage.googleapis.com/cocalc-extra/cocalc-latex-editor-2019.png").fit
       div.col-sm-6
         p.
-          Collaboratively #[strong edit #[a(href="https://www.latex-project.org/") LaTeX documents] right inside your browser].
+          Collaboratively #[strong edit #[a(href="https://www.latex-project.org/") LaTeX documents] right inside your browser],
+          without a limit on the number of collaborators!
```

Comment:
The whole point is that our FREE VERSION supports an unlimited number of collaborators, specifically unlike overleaf, whose **free version** only supports one collaborator.   Overleaf also supports unlimited collabs... if you pay them.  So I think the wording above kind of misses the point.